### DISCUSSION TITLE
无脑反共的智障真多

### DISCUSSION THREAD

**Discussion by wangsir003** [WeNeedHome/SummaryOfLoanSuspension#1179] [Ideas]
这些反共的，总结他们的逻辑就是没有言论自由，压榨；试问没有中工你们现在还有能力在这儿发言？早被日美英奴役了吧

**Comment by mand-byte** [comment]
仇恨教育的脑瘫儿

**Reply by maxwillma** [reply]
中二 就这样

**Comment by wedxz** [comment]
你这话能在国内发布？你这是拿着自由反自由

**Comment by JayChou55** [comment]
按你这逻辑 台湾同胞 早应该被欧美奴役的跟狗一样了 比较没有中g的领导统治

**Reply by Cegazhos** [reply]
谁几把跟你同胞啊 傻狗东西

**Reply by b4o** [reply]
台积电过不了几年，就成美国的了，看看台湾领导见到美国爸爸，那奴颜屈膝 唯唯诺诺的样子，真是狗都不如啊

**Reply by JayChou55** [reply]
二级管是真的蠢。。。懒得辩，只想笑

**Comment by itsoursduty** [comment]
“总结他们的逻辑就是没有言论自由，压榨” 你就说这些还够不够 ，该不该反

**Comment by frshgrass** [comment]
鉴定为煞笔 日常遭遇不公只敢唯唯诺诺的垃圾人

**Reply by jiaojiaozhe123456** [reply]
不，这就是骑在我们脖子上的人

**Comment by Cegazhos** [comment]
反中修是好事，但他们没有脑子。脑子里只想给台湾叫委屈的傻狗，就是被人操高潮了也惦记着要给台湾引流😅

**Comment by gkrudtkfkd** [comment]
你是核酸做少了！ 回去在做2次！言论不自由也行！让法律能维护正义我们就闭嘴！我只求法制

**Reply by zdcaily** [reply]
哈哈哈，看来美国200多万人白死了

**Comment by lance111l** [comment]
自由是有边界的   特朗普的账号都能封  你算啥  
只有与媒体一致时 才自由

**Comment by wanliheshan123** [comment]
你说得对 但是十八大之后抓了两百以上中共官员

**Reply by dinglino1** [reply]
哪有那么多贪官，不过就是内斗二医

**Comment by Hcreak** [comment]
你可真是个傻逼

**Comment by ghost** [comment]
> 这些反共的，总结他们的逻辑就是没有言论自由，压榨；试问没有中工你们现在还有能力在这儿发言？早被日美英奴役了吧

为什么一个党自称共产党就叫共产党！
还有，我们的美好生活是“共产党”赐予的，还是你父母祖辈烈士你自己拼出来的。
强烈建议分清：country state homeland。

**Comment by kangzejian1896** [comment]
如果没有共产党，我们就过上新加坡、台湾的生活了，好可怕！
